documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -123,13 +123,21 @@ Add the following to your Gradle build file in order to enable MapStruct:
 ...
 plugins {
     ...
-    id 'net.ltgt.apt' version '0.8'
+    id 'net.ltgt.apt' version '0.15'
 }
+
+// You can integrate with your IDEs.
+// See more details: https://github.com/tbroyer/gradle-apt-plugin#usage-with-ides
+apply plugin: 'net.ltgt.apt-idea'
+apply plugin: 'net.ltgt.apt-eclipse'
+
 dependencies {
     ...
     compile 'org.mapstruct:mapstruct-jdk8:{mapstructVersion}'
 
-    apt 'org.mapstruct:mapstruct-processor:{mapstructVersion}'
+    annotationProcessor 'org.mapstruct:mapstruct-processor:{mapstructVersion}'
```

Comment:
It should be
```
    annotationProcessor "org.mapstruct:mapstruct-processor:{mapstructVersion}"
    testAnnotationProcessor "org.mapstruct:mapstruct-processor:${mapstructVersion}"
```
double quotes `"` instead of single quotes `'`
`${mapstructVersion}` instead of `{mapstructVersion}`

Reply:
@jakraska Thanks! I'll modify it